quickstarts/mean/config.yml

```diff
@@ -0,0 +1,52 @@
+
+slug: mean-stack
+description: |
+  ## Comprehensive MEAN stack monitoring system
+
+  MEAN stack is an open source JavaScript-based web application development framework for building dynamic websites and web applications. The stack consists of MongoDB, ExpressJS, Angular and NodeJS.
+  Since all the components are based on JavaScript and JSON, the integration between the components of the stack is intuitive and straightforward.
+  Each of these 4 powerful technologies provides an end-to-end framework for the developers to work in and each of these technologies plays a big part in the development of web applications.
+
+  ## What should you look for in a MEAN dashboard?
+  - Metrics of MongoDB, i.e. number of active connections, database details, usage time and many more.
+  - Metrics of ExpressJS, NodeJS, i.e. number of web transactions, top 10 transactions and the transactions errors, memory usage and many more.
+  - Metrics of Angular, i.e. number of viewed urls, page views by city, ajax requests and many more. You are also free to create your own chart by using the stored metrics.
+
+  ## What’s included in the MEAN quickstart?
+  Install this quickstart to install preconfigured observability solutions:
+  - Multiple high-value alerts  including average number of created connections, FCP score, memory usage and more.
+  - Informative dashboards (Top 10 transactions, connections, database indexes and more).
+
+summary: |
+  MEAN is named for the four major technologies that comprise the stack's layers: MongoDB, Express, Angular, and NodeJS.
+  Our MEAN integration with the Browser monitoring agent(Angular), Database monitoring agent(MongoDB) and Application Monitoring agent(Nodejs), lets you monitor your apps with our quickstart dashboards right out of the box.
+
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: MEAN stack
+documentation:
+  - name: MEAN stack installation documentation
+    description: |
+      Integrate your MEAN app and get your data into New Relic.
+    url: >- 
+      https://docs.newrelic.com/docs/browser/browser-integrations/mexn-integration/
+keywords:
+  - MEAN
+  - MongoDB
+  - database
+  - ExpressJS
+  - Angular
+  - NodeJS
+  - Fullstack
+installPlans:
+  - mongodb-integration
+  - node-agent
+  - browser-docs
+dataSourceIds:
+  - mean-stack
+dashboards:
+  - mexn
+alertPolicies:
```

Comment:
Seems like you haven't made an alert with this id of mexn. Either create a dashboard or reference an already existing dashboard via id.